examples/dns/package.json

```diff
@@ -26,12 +26,12 @@
     "url": "https://github.com/open-telemetry/opentelemetry-js/issues"
   },
   "dependencies": {
-    "@opentelemetry/api": "^0.15.0",
-    "@opentelemetry/exporter-jaeger": "^0.15.0",
-    "@opentelemetry/exporter-zipkin": "^0.15.0",
-    "@opentelemetry/node": "^0.15.0",
+    "@opentelemetry/api": "^0.16.0",
+    "@opentelemetry/exporter-jaeger": "^0.16.0",
+    "@opentelemetry/exporter-zipkin": "^0.16.0",
+    "@opentelemetry/node": "^0.16.0",
     "@opentelemetry/plugin-dns": "^0.13.0",
```

Comment:
Is the update to 0.14.0 a followup PR?

Reply:
I was going to cut a 0.14.0 release after this PR

Reply:
devDependencies are pinned so the CI is faster. prod dependencies are ranged so customers can receive important bugfixes without manual intervention.